Source/WebKit/WebProcess/InjectedBundle/mac/InjectedBundleMac.mm

```diff
@@ -241,7 +241,7 @@ static NSEventModifierFlags currentModifierFlags(id self, SEL _cmd)
     m_classesForCoder = mutableSet;
 }
 
-NSSet* InjectedBundle::classesForCoder()
+RetainPtr<NSSet> InjectedBundle::classesForCoder()
```

Comment:
Presumably `return m_classesForCoder.get();` below can become `return m_classesForCoder;` now? Also seems like maybe it could use lazyInitialize, but not sure what other constraints it's under.